x-pack/plugin/security/qa/security-trial/src/javaRestTest/java/org/elasticsearch/xpack/security/failurestore/FailureStoreSecurityRestIT.java

```diff
@@ -2027,6 +2027,84 @@ public void testFailureStoreAccessWithApiKeys() throws Exception {
         expectThrowsWithApiKey(apiKey, new Search("test1"), 403);
     }
 
+    public void testFieldCapabilities() throws Exception {
```

Comment:
Can be in a follow up, but some basic coverage of field-caps + FLS would also be great.

Reply:
Good suggestion. I've pushed https://github.com/elastic/elasticsearch/pull/126210/commits/0a436c0f1cd19a985d06be13f7f3ae19ad73748a